src/components/page/blocks/WidgetBlock.jsx

```diff
@@ -6,6 +6,7 @@ import GalaxySelector from '../../../containers/GalaxySelectorContainer.jsx';
 import GalaxiesSelector from '../../../containers/GalaxiesSelectorContainer.jsx';
 import HubblePlot from '../../../containers/HubblePlot2DContainer.jsx';
 import HubblePlot3D from '../../../containers/HubblePlot3DContainer.jsx';
+import HeatMap3DPlot from '../../../containers/HeatMap3DPlotContainer.jsx';
```

Comment:
Please rename this Class and filename to something like LargeScaleStructureSOMETHING instead of HeatMapSOMETHING as these are not heat maps, and we may have other components in the future that are.